src/content/docs/cypher/query-clauses/load-from.md

```diff
@@ -74,7 +74,12 @@ RETURN age, name LIMIT 3;
 ```
 
 ### Enforce Schema
-To enforce a specific schema and data types when reading from CSV, you can use the `LOAD WITH HEADERS (<name> <dataType>, ...) FROM ...` syntax.
+By default, Kùzu will infer the column names and data types from the scan source automatically, 
+- for parquet, pandas, polars & pyarrow, column names and data types are always available in the data source.
+- for csv, we use header as schema if available, otherwise we fallback naming to `column0, column1, ...` and assume all data types are `STRING`
```

Comment:
```suggestion
- For CSV, we use header as schema if available, otherwise we fallback naming to `column0, column1, ...` and assume all data types are `STRING`
```